gatsby-config.ts

```diff
@@ -31,7 +37,7 @@ module.exports = {
       resolve: `gatsby-plugin-intl`,
       options: {
         // language JSON resource path
-        path: `${__dirname}/src/intl`,
```

Comment:
One of the current limitations where `__dirname` will not work as expected. More on this here: https://www.gatsbyjs.com/docs/how-to/custom-configuration/typescript/#__dirname